app/models/notification.rb

```diff
@@ -136,6 +136,20 @@ def update_notifications(notifiable, action = nil)
       Notifications::UpdateJob.perform_later(notifiable.id, notifiable.class.name, action)
     end
 
+    def fast_destroy_old_notifications(destroy_before_timestamp = 4.months.ago)
```

Comment:
I would move this in a service object, we already have a `Notifications::RemoveAll` and a `Notifications::RemoveAllByAction` in the `service/notifications` folder, maybe we can add a third object called `Notifications::BatchDelete` or something like that.

BTW I tried to execute this from the console and I get:

> WARNING:  there is no transaction in progress

It might be because in the `BulkSqlDelete` we run commit, but not `begin_db_transaction`

Reply:
I think this will likely end up being used for other objects which is why I kept the class name unspecific.

Reply:
yeah, I didn't mean to rename `BulkSqlDelete`, just to move it as a service object. 

The comment about naming was related to `fast_destroy_old_notifications` which I think should be a service object as well.